### PR TITLE
New AcsRouterWorker events

### PR DESCRIPTION
**Pull request by billwert**

### CODE REVIEW

File: sdk/eventgrid/azure-messaging-eventgrid/src/main/java/com/azure/messaging/eventgrid/implementation/models/AcsAdvancedMessageChannelEventError.java [hunk 1]
```diff
@@ -0,0 +1,112 @@
...
+import com.azure.json.JsonSerializable;
+import com.azure.json.JsonToken;
+import com.azure.json.JsonWriter;
+import java.io.IOException;
+
+/**
+ * Advanced Message Channel Event Error.
+ */
+@Fluent
+public final class AcsAdvancedMessageChannelEventError
```
Comment (srnagar) [inline sdk/eventgrid/azure-messaging-eventgrid/src/main/java/com/azure/messaging/eventgrid/implementation/models/AcsAdvancedMessageChannelEventError.java:18]: Have we tried using the error type defined in the common-types swagger?https://github.com/Azure/azure-rest-api-specs/blob/main/specification/common-types/data-plane/v1/types.json

If the swagger is updated to use this type, then the generator will use ResponseError.

**Review by srnagar — APPROVED** [review]

File: sdk/eventgrid/azure-messaging-eventgrid/src/main/java/com/azure/messaging/eventgrid/implementation/models/AcsAdvancedMessageChannelEventError.java (hunk 1 above)
Comment (billwert) [reply sdk/eventgrid/azure-messaging-eventgrid/src/main/java/com/azure/messaging/eventgrid/implementation/models/AcsAdvancedMessageChannelEventError.java:18]: discussed offline, this one is going to stay as is.

**Review by JoshLove-msft — APPROVED** [review]

**Pull request merged**